_sass/pages/_who_uses_login.scss

```diff
@@ -0,0 +1,92 @@
+.container.who-uses-login {
+  .partners {
+    background: url(../img/who-uses-login/who-uses-illo-partners/who-uses-illo-partners.svg) left center no-repeat;
+    padding: 3rem 0 3rem 40%;
+  }
+
+  .security {
+    display: flex;
+    padding: 3rem 0;
+
+    > p {
+      flex-basis: 0;
+      flex-grow: 1;
+
+      &:first-child {
+        background: url(../img/who-uses-login/icon-shield/icon-shield.svg) left top no-repeat;
+        padding: 0 0 0 3rem;
+      }
+
+      &:last-child {
+        background: url(../img/homepage/icon-partner/icon-partners.svg) 0.4rem -.5rem no-repeat;
+        border-left: 1px solid darken($blue-lightest, 15%);
+        margin-left: 0.25rem;
+        padding: 0 0 0 4rem;
+      }
+    }
+  }
+
+  .mobile {
+    display: none;
+  }
+}
+
+.who-uses-login ~ .one-account-for-govt {
+  margin-top: 0;
+  margin-bottom: 5rem;
```

Comment:
Is this something where the footer should be applying a default top margin instead? Feels like that's going to be common across pages.

Reply:
I think that is a great idea, but will take us re-assessing all of the pages on the site and seeing how that margin will effect the content on those pages. Let's leave as is for now and as we do the final QA, assess then.

Going to leave as is for now.